### ISSUE TITLE
Feature request: Allow custom `Tint Color` in combination with `Match System`

### ISSUE THREAD

**Issue by fooness** [Dimillian/IceCubesApp#1666]
- [x] Search that this request don't already exist before creating it.

As of right now, it’s seemingly not possible to choose `Match System` in the display/theme settings, **and also** set a custom `Tint Color` for all across the application. It would be *very* great to allow for this customization.

The `Match System` option is very convenient in case the system setting is set to automatically apply a light theme during the day, and a dark theme during the night. This feature cannot be “emulated” in Ice Cubes, hence it depends on the `Match System` option being checked. Not being able to choose a custom `Tint Color` in this case, though, is not really good—also probably from an accessibility point of view.

**Comment by AntoineKemp** [comment]
Indeed. I put links in orange, but once I quit IceCubes and log in again, the orange Tint Color is gone, all is black again (Match Syste, set). The color circle is completely black. See screenshot.
![IceCubes](https://github.com/Dimillian/IceCubesApp/assets/149075737/d81cbea5-8748-4d03-8eb3-fca28c2b360a)

**Comment by thedeany** [comment]
I would really love to see this implemented as well. I like the default purple, but I'd prefer to choose my own color while retaining the Match System ability.
